truffle/com.oracle.truffle.api.debug/src/com/oracle/truffle/api/debug/Debugger.java

```diff
@@ -957,15 +957,9 @@ public FrameInstance visitFrame(FrameInstance frameInstance) {
                     if (stackIndex < contextStackDepth) {
                         final Node callNode = frameInstance.getCallNode();
                         if (callNode != null) {
```

Comment:
Yes I think so. If your code works with the latest Truffle that should be the behaviour from now on. It was a bug that you weren't being given these frames with `null` `CallTarget`s before.